Sources/BraintreeVenmo/BTVenmoAppSwitchRequestURL.swift

```diff
@@ -0,0 +1,104 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+// TODO: Entire class be internal and likely a struct once rest of Venmo is in Swift
+// TODO: methods can all be non-static likely when we convert the rest of Venmo to Swift
+@objcMembers public class BTVenmoAppSwitchRequestURL: NSObject {
```

Comment:
Total 🏈 question - we use the phrases "redirect URL", "request URL", and "return URL" mixed throughout the SDK. Would be good to only use 2 and clearly define which is used for switching TO the app and which is for switching FROM the app.

Reply:
Yeah, these will be internal once the client is in Swift, so updating the names to be more clear makes sense to me. What names do you prefer?

Reply:
Chatted IRL - going forward with "redirectURL" to represent the URL used to switch _to_ the checkout flow from a merchant app, and "returnURL" when coming back to the merchant app _from_ the web view.

Reply:
Note for self to update the error case `BTVenmoErrorTypeInvalidRequestURL` in follow up client PR